pages/_app.tsx

```diff
@@ -109,9 +112,18 @@ CustomApp.getInitialProps = async (
     ? 'development'
     : 'production';
 
+  const appConfig = {
```

Comment:
It feels like it's probably worth a comment here warning that the values put in this object are passed to the client app, and therefore are publicly available in the browser. Maybe:

```suggestion
  // Warning! Any values placed in this object are passed to the client app, and will be made public in the browser. Do not share any sensitive or secret  values through this.
  const appConfig = {
```

Reply:
Probably best to add a similar warning to the docs you've written too!